data/json/npcs/refugee_center/surface_refugees/NPC_Dino_Dave.json

```
@@ -50,7 +73,53 @@
     "dynamic_line": "*gestures about his new home.  \"We have everything we wanted.  They cannot see us here, and our mind is at peace.  We go outside as we need, and the safety we have received here stays with us, as long as we are careful about it.  In time, perhaps we will again have need of your noble services.\"",
     "responses": [
       { "text": "You seem different.", "topic": "TALK_DINODAVE_Different" },
+      {
+        "text": "You know, this place is surprisingly nice, for a cardboard fortress.",
+        "topic": "TALK_DINODAVE_Cardboard",
+        "effect": [
+          { "u_add_var": "u_cardboard_gift", "type": "dialogue", "context": "dave", "value": "in_progress" },
+          { "math": [ "u_timer_dave_creating_gift", "=", "time('now')" ] }
+        ],
+        "condition": {
+          "and": [
+            { "not": { "u_has_var": "u_cardboard_gift", "type": "dialogue", "context": "dave", "value": "finished" } },
+            {
+              "not": { "u_has_var": "u_cardboard_gift", "type": "dialogue", "context": "dave", "value": "in_progress" }
+            }
+          ]
+        }
+      },
       { "text": "Always good to catch up, Dave.  Keep on being you.", "topic": "TALK_DONE" }
     ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_DINODAVE_Cardboard",
+    "dynamic_line": "* bows deeply.  \"Thank you.  Our expertise with the most sacred of materials comes from a past long gone.\"  He gazes off into the distance, lost in thought.  You wait, but he doesn't talk again.",
```

```suggestion
    "dynamic_line": "*bows deeply.  \"Thank you.  Our expertise with the most sacred of materials comes from a past long gone.\"  He gazes off into the distance, lost in thought.  You wait, but he doesn't talk again.",
```
`*` is used like so in previous instances so...